pr-data.csv

```diff
@@ -4223,7 +4223,7 @@ https://github.com/google/guava,a197d99b96d69ea75a2c4a15ecd8a730dd3270d6,guava-t
 https://github.com/google/guava,a197d99b96d69ea75a2c4a15ecd8a730dd3270d6,guava-testlib,com.google.common.collect.testing.testers.MapToStringTester.testToStringWithNullKey[setUpTearDown [collection size: several]],ID,,,
 https://github.com/google/guava,a197d99b96d69ea75a2c4a15ecd8a730dd3270d6,guava-test,com.google.common.collect.testing.testers.MapToStringTester.testToStringWithNullKey[synchronized ArrayListMultimap [collection size: several].asMap [collection size: several]],ID,,,
 https://github.com/google/guava,a197d99b96d69ea75a2c4a15ecd8a730dd3270d6,guava-test,com.google.common.collect.testing.testers.MapToStringTester.testToStringWithNullKey[Synchronized.setMultimap [collection size: several] reserialized [collection size: several].asMap [collection size: several]],ID,,,
-https://github.com/google/guava,a197d99b96d69ea75a2c4a15ecd8a730dd3270d6,guava-test,com.google.common.collect.testing.testers.MapToStringTester.testToStringWithNullKey[Synchronized.setMultimap [collection size: several].asMap [collection size: several]],ID,,,
+https://github.com/google/guava,a197d99b96d69ea75a2c4a15ecd8a730dd3270d6,guava-test,com.google.common.collect.testing.testers.MapToStringTester.testToStringWithNullKey[Synchronized.setMultimap [collection size: several].asMap [collection size: several]],ID,MovedOrRenamed,,https://github.com/google/guava/commit/ae2c37b1ba5de61df02e1f018e294f255c071c32
```

Comment:
This test was reported in the commit `a197d99b96d69ea75a2c4a15ecd8a730dd3270d6` from Dec 1, 2021. Your change claims that this test was moved in the commit `ae2c37b1ba5de61df02e1f018e294f255c071c321`. However, that commit is from May 22, 2013. We do not care about the moves that happened before the test was reported but care only about the move(s) that happened after the test was reported.

I still think it's better to remove this line from this PR and add in some future PR.

Reply:
Thanks for pointing it out. I researched a bit further and got these results:

If you look at the repo from this commit (the commit when the test was reported )- https://github.com/google/guava/tree/a197d99b96d69ea75a2c4a15ecd8a730dd3270d6): [commit tree](https://github.com/google/guava/tree/a197d99b96d69ea75a2c4a15ecd8a730dd3270d6)

The module guava-test is not available, and if I try to find the MapToStringTester.java file in this, it clearly shows that it's only present under guava-testlib module and not under guava-test. The MapToStringTester.java was indeed moved under guava-testlib in this commit `ae2c37b1ba5de61df02e1f018e294f255c071c321` and it hasn't been moved from it after that. 

I will also split this PR as per your suggestion. Keeping this comment here as a tracker for this conversation.

Thanks.